### DISCUSSION TITLE
is -forceupdate flag working?

### DISCUSSION THREAD

**Discussion by HairyMilkshakes** [Pryaxis/TShock#2033] [Q&A]
Ran a server for about 30 minutes with it enabled and didn't see any growth progress. Anybody else experiencing the same issue?

**Comment by Kojirremer** [comment]
I'm pretty sure it didn't work on 1.3.x. Haven't tried the 1.4 builds.

**Reply by luoyianwu** [reply]
Same here, running v1.4.0.5.

**Comment by hakusaro** [comment]
Fairly sure that it's broken these days, sadly.
